### PR TITLE
w3emc: add v2.11.0

### PR DESCRIPTION
**Pull request by edwardhartnett**
added version 2.11.0 for package w3emc

### CODE REVIEW

File: var/spack/repos/builtin/packages/w3emc/package.py
Comment (edwardhartnett) [inline var/spack/repos/builtin/packages/w3emc/package.py:60]: @AlexanderRichert-NOAA should these be parens or square brackets?

**Review by Hang-Lei-NOAA — APPROVED** [review]
This is fine.

**Comment by alecbcs** [comment]
Looks good to me. Will wait for @AlexanderRichert-NOAA's response to the question above before merging.

File: var/spack/repos/builtin/packages/w3emc/package.py
Comment (AlexanderRichert-NOAA) [reply var/spack/repos/builtin/packages/w3emc/package.py:60]: Doesn't technically matter, but some Spack devs have been pushing for using tuples (the parens) when possible, so I'd say let's use parens for consistency.

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Review by alecbcs — APPROVED** [review]
Confirmed SHA. @edwardhartnett thanks for the update.

**Pull request merged**
